int8 command
---
int8 command_res
